### PR TITLE
Add favorites panel to landing page

### PR DESCRIPTION
**Pull request by InsaneZein**

### CODE REVIEW

**Review by Hyperkid123 — COMMENTED** [review]
I am having a issue when running locally.

Seems like some of the components within the `LandingNavFavorites` use internationalization. And it is not set up on the landing page.

We have 3 options
- find the child node that needs the provider and remove any usage if react-intl (not something I would recommend)
- Add the IntlProvider to Landing page tree and configure missing messages
- Share react-intl as a singleton (this can be dangerous)


screenshot-stage foo redhat com_1337-2023 05 17-08_30_28

**Comment by InsaneZein** [comment]
hmm sounds like the second option might be the best

File: src/App.js [hunk 1]
```diff
@@ -37,30 +36,29 @@ const App = () => {
   chrome?.updateDocumentTitle?.('Hybrid Cloud Console Home', false);
 
   useEffect(() => {
+    console.log(__webpack_share_scopes__.default);
```
Comment (Hyperkid123) [inline src/App.js:39]: Sorry, this is a left over from the debugging I was doing. It does not belong here.

**Review by Hyperkid123 — APPROVED** [review]

**Comment by Hyperkid123** [comment]
We will have to wait with promoting this to production for the chrome to get promoted first.

**Pull request merged**